metadata-ingestion/tests/unit/test_airflow.py

```diff
@@ -19,9 +19,9 @@
 
```

Comment:
You may want to store tests for operators, hooks, and the lineagebackend in datahub_provider like done in the sample repo: https://github.com/astronomer/airflow-provider-sample/tree/main/tests. This way modules from datahub_provider won't need to be imported into datahub.

Reply:
Not sure I'm following the suggestion - the tests in both the sample repo and this one are stored separate from the main source directories.

The `datahub` imports of `datahub_provider` are primarily for backwards compatibility, so I'm pretty ok with leaving those in the code.

Reply:
That makes sense was just suggesting because module tests are usually packaged with modules.